staging/src/k8s.io/code-generator/cmd/deepcopy-gen/output_tests/aliases/doc.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 // +k8s:deepcopy-gen=package
 
 // This is a test package.
-package aliases
+package aliases // import "k8s.io/code-generator/cmd/deepcopy-gen/output_tests/aliases"
```

Comment:
Do we still need this comment?

I had the impression that it became obsolete with the introduction of Go modules (which also define the import path), but perhaps I am wrong?

Reply:
I still don't have a full understanding for the staging repos but I think go toolchain will not handle custom imports ? (we use `replace` in the `go.mod` file).
We either drop this practice or keep it consistent over the entire code base: https://grep.app/search?q=//%20import%20%22k8s.io

Reply:
What do you mean with "custom imports"?

> We either drop this practice

I think that's the path that we should take - but let's be sure that it's really not needed anymore.

Reply:
Let's use `k8s.io/dynamic-resource-allocation` as example as it does not have the `// import` comment. As I expected, Go complains when it gets imported using the wrong name.

https://go.dev/play/p/69zPbXw7mFx

```golang
import (
	"fmt"

	"github.com/kubernetes/dynamic-resource-allocation/kubeletplugin"
)
```

```
go: finding module for package github.com/kubernetes/dynamic-resource-allocation/kubeletplugin
go: downloading github.com/kubernetes/dynamic-resource-allocation v0.31.3
go: found github.com/kubernetes/dynamic-resource-allocation/kubeletplugin in github.com/kubernetes/dynamic-resource-allocation v0.31.3
go: play imports
	github.com/kubernetes/dynamic-resource-allocation/kubeletplugin: github.com/kubernetes/dynamic-resource-allocation@v0.31.3: parsing go.mod:
	module declares its path as: k8s.io/dynamic-resource-allocation
	        but was required as: github.com/kubernetes/dynamic-resource-allocation

Go build failed.
```

Reply:
Yeah, modules document the canonical import path. I don't think we still need these.